docs/requirements-doc.txt

```diff
@@ -13,3 +13,5 @@ git+https://github.com/modin-project/modin.git@master#egg=modin[all]
 sphinxcontrib_plantuml
 sphinx-issues
 xgboost
+# TODO: remove protobuf dependency when ray-project resolves GH#25205
+protobuf<4.0
```

Comment:
Is this required here? We install `git+https://github.com/modin-project/modin.git@master#egg=modin[all]` above so a pin for protobuf in setup.py should be enough.

Reply:
Removed!

Reply:
It is strange that protobuf > 4.0 is installed for build docs.